sasmodels/models/superball.py

```diff
@@ -20,6 +20,7 @@
 with $a$ the cube edge length of the superball and $p$ a parameter that
 describes the roundness of the edges. In the limiting cases $p=1$ the superball
 corresponds to a sphere with radius $R = a/2$ and for $p = \infty$ to a cube
+
```

Comment:
A bunch of blank lines have been introduced and should be removed. They are mostly harmless, but this one will mess up the formatting.

Reply:
The extra blank lines have been cleaned. No idea where they came from...